src/Viewer/renderers/mathInput.jsx

```diff
@@ -132,7 +134,8 @@ export default function MathInput(props) {
   const handleFocus = (e) => {
     setFocusedField(() => handleVirtualKeyboardClick);
     setFocusedFieldReturn(() => handlePressEnter);
-    setFocusedFieldID(mathField.id);
+    // setFocusedFieldID(mathField.id);
```

Comment:
Removed this to style the focused mathInput-- but it seems important for the keyboard.